src/main/java/com/nology/synergyapi/controller/UserContactsController.java

```diff
@@ -19,14 +21,16 @@
 public class UserContactsController {
     @Autowired
     UserContactsService userContactsService;
+    @Autowired
     UserContactsRepository userContactsRepo;
+    @Autowired
+    private UserRepository userRepository;
 
 
-
```

Comment:
With the updated model, UserContact model will be deleted as the table will be generated already from the User model.   
If the objective here is to get all possible contacts, then I suggest we create a getAllContactsWithBank in UserContactsService similar to getUserContactsWithBank (and contacts endpoint).   The only difference is that instead of looping through a specific user's contacts, we will need to loop through all records in the user table.